File: tasks/2024/round1_pp/kerekerdo.py
#!/usr/bin/env python3
# @check-accepted: examples N5 N100 full

n, a, b = list(map(int, input().strip().split()))
fak = (b-a+1 if a<=b else n-(a-b)+1)
print("Tul konnyu!" if fak<5 else fak)

File: tasks/2024/round1_pp/kerekerdo.cpp
// @check-accepted: examples N5 N100 full
#include <iostream>
using namespace std;

int main() {
    int n,a,b;
    cin>>n>>a>>b;
    int fak;
    if(a<=b) {
        fak=b-a+1;
    }else {
        fak=n-a+b+1;
    }

    if(fak<5) {
        cout<<"Tul konnyu!\n";
    }else {
        cout<<fak<<"\n";
    }

    return 0;
}
